src/py42/clients/alerts.py

```diff
@@ -42,23 +45,44 @@ def get_details(self, alert_ids):
     def resolve(self, alert_ids, reason=None):
         """Resolves the alerts with the given IDs.
 
+        Deprecated. Support will be removed from version 2.0.0. Use `update_state` instead.
+
         Args:
             alert_ids (iter[str]): The identification numbers for the alerts to resolve.
             reason (str, optional): The reason the alerts are now resolved. Defaults to None.
 
         Returns:
             :class:`py42.response.Py42Response`
         """
-        return self._alert_service.resolve(alert_ids, reason=reason)
+        return self._alert_service.update_state(
+            AlertState.DISMISSED, alert_ids, reason=reason
+        )
 
     def reopen(self, alert_ids, reason=None):
         """Reopens the resolved alerts with the given IDs.
 
+        Deprecated. Support will be removed from version 2.0.0. Use `update_state` instead.
```

Comment:
Can you use a doc str link for `update_state` ?

`:meth:\`~py42.clients.alerts.AlertClient.update_state()\``

Reply:
Replace &#96 with backtick

Reply:
Its already part of AlertClient doc, do we need a separate one?

![image](https://user-images.githubusercontent.com/61223509/92608909-89ac1980-f2d3-11ea-90fe-7579db656e88.png)

Reply:
ohh, ignore, I see what you mean.

Reply:
Updated.